docs/commands/lint.md

```diff
@@ -2,239 +2,378 @@
 
-Instead of full paths, you can use names listed in the `apis` section of your Redocly configuration file as entrypoints.
+`@version` is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the `apis` section must be unique.
 
-```bash Command
-openapi lint main
-```
+You can pass `name` or `name@version` in commands like `lint` and `stats` to act only on those APIs. Below is an example `apis` section showing three files identified using `main`, `production` and `sandbox`:
 
 ```yaml Configuration file
 apis:
   main:
-    root: ./openapi/definition.json
+    root: ./openapi.yaml
+    labels:
+      - main
+    lint: []
+    features.openapi: []
+  production:
+    root: ./production.yaml
+    labels:
+      - production
+    lint: []
+    features.openapi: []
+  sandbox:
+    root: ./sandbox.yaml
+    labels:
+      - sandbox
+    lint: []
+    features.openapi: []
 ```
 
-In this case, after resolving the path behind the `main` name (see the `Configuration file` tab), `lint` will validate the `definition.json` file. The presence of the Redocly configuration file is mandatory.
+ In this example, we only want to validate files under `main`:
 
-#### Empty entrypoints
+```bash Command
+redocly lint main
+```
 
-You can omit entrypoints completely when executing the `lint` command.
+##### Method 3: Pass no entrypoints
 
-```bash Command
-openapi lint
+Running `lint` without specifying a particular API definition file checks all definition files in your project directory.
+
+```bash
+redocly lint
 ```
 
-```yaml Configuration file
-apis:
-  main:
-    root: ./openapi/definition.json
-  production:
-    root: ./openapi/production.yaml
-  sandbox:
-    root: ./openapi/sandbox.yaml
+:::warning Important
+If you run `redocly lint` without entrypoints, and your project has no configuration files, you'll get an error.
+:::
+
+### How to reference a custom config file
+
+By default, Redocly CLI looks for the [Redocly configuration file](/docs/cli/configuration/configuration-file.mdx) in the current working or project directory. Using the optional `--config` argument provides an alternative path to a different configuration file if one exists.
+
+```bash
+redocly lint --config=./another/directory/anotherconfig.yaml
 ```
 
-In this case, if no API definitions are specified, `lint` validates all entrypoints listed under `apis` in your Redocly configuration file. The presence of the configuration file is mandatory.
+### How to activate rule sets
 
-:::warning Important
+Redocly CLI comes with [built-in rules](./resources/built-in-rules.md) that are used to validate API definitions. If you want to override a rule or reduce its severity, you can do so by modifying `redocly.yaml` (globally or per API definition).
```

Comment:
```suggestion
Redocly CLI comes with [built-in rules](./resources/built-in-rules.md) that are used to validate API definitions.
```
That sentence feels misplaced. The built-in rules aren't enabled automatically. The user can enable or disable them in their configuration file.